docs/source/algorithms/PowderDiffILLDetEffCorr-v1.rst

```diff
@@ -9,84 +9,196 @@
 
-**Example - PowderDiffILLDetEffCorr**
+  - 0 stands for auto: Iterations will be run automatically until the termination criteria is satisfied. Termination criteria is:
+
+      .. math:: \chi^2/NdoF = \frac{\sum_{i,j}(c_{ij} - 1)ˆ2}{N_{tubes} * N_{pixels_per_tube}} < t
+
+      where :math:`c_{ij}` is the residual calibration factor for tube *i* and pixel *j*, :math:`t` is the threshold defined in :ref:`Instrument Parameter File (IPF)<InstrumentParameterFile>` as *chi2_ndof*.
+
+      The top and bottom parts of the tubes are excluded from this calculation. How many pixels exactly are excluded is again defined in :ref:`IPF <InstrumentParameterFile>` as *pixels_to_trim*.
+
+      Currently, for **D2B**, *pixels_to_trim=28* and *t=1%*. With this settings iterations typically terminate after the first one, i. e. one run and one iteration give results already convergent within 1%.
+
+      This has to be interpreted as: the residual calibration is close enough to unity, so further iterations will not change the calibration much.
+
+      However, this criterion does not prevent from divergence in all the cases.
+      It can happen that for a given pixel the residual calibration factor (albeit close to unity) is always on the same side (i.e. always above 1 or below 1); this will cause the absolute calibration to gradually diverge with iterations.
+      Anyways, the method implemented does not provide enough precision to resolve residual calibration better than in the percent range.
+      Hence, care must be taken when using the iterations.
+      It is not recommended to use more than 2 iterations.
+
+Limitations
+-----------
+
+For **D2B** it is assumed that the tubes and pixels pass through the exact same positions during the scan.
+That is, the tubes have to be aligned vertically and horizontally and the gap between each pair of neighboring tubes must be integer multiple of the scan step.
+
+D20 Workflow
+------------
+
+.. diagram:: PowderDiffILLDetEffCorr-v1_D20_wkflw.dot
+
+**Example - D20**
 
 .. code-block:: python
 
-   calib = PowderDiffILLDetEffCorr(CalibrationRun='967076', OutputWorkspace='constants')
-   print("Reduced workspace contains {0} constants, one for each cell.".format(calib.getNumberHistograms()))
+   import matplotlib.pyplot as plt
+   from mantid import plots
+   from mantid.simpleapi import PowderDiffILLDetEffCorr
+   PowderDiffILLDetEffCorr(CalibrationRun='967076.nxs', DerivationMethod='SequentialSummedReference1D', OutputWorkspace='calib')
+   Transpose(InputWorkspace='calib', OutputWorkspace='calib')
+   fig, ax = plt.subplots(subplot_kw={'projection':'mantid'})
+   ax.plot(mtd['calib'],'-')
+   ax.set_xlabel('Pixel #')
+   ax.set_ylabel('Calibration constant')
+   fig.show()
+
+.. figure:: /images/D20_calib.png
```

Comment:
Could the `.. plot:` directive be used here or does it take too long to produce the documentation?